google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -845,6 +848,15 @@ public static SetStorageClassLifecycleAction newSetStorageClassAction(
         return new SetStorageClassLifecycleAction(storageClass);
       }
 
+      /**
+       * Create a new {@code AbortIncompleteMPUAction}. An incomplete multipart upload will be
+       * aborted when the multipart upload meets the specified condition. Age is the only condition
+       * supported for this action. See: https://cloud.google.com/storage/docs/lifecycle##abort-mpu
+       */
+      public static LifecycleAction newAbortIncompleteMPUploadAction() {
```

Comment:
please add unit tests as well

Reply:
done, I also added some additional checks for the existing lifecycle action tests.